config/initializers/devise.rb

```diff
@@ -0,0 +1,312 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '2b18bbd5627f961708ce71b5d181c2522ad3ade569e3ee11141f0644b49b763a80dc11d6aab5c16fd6e8beb895687f32c411dcdb092008842691c2cc6a4ddb10'
```

Comment:
We should put this data in a separate file called `.env` that doesn't go into GitHub and import it into this file.

Reply:
If you'd prefer, we could wait to make this change until a later PR since there's no actual secret key yet.

Reply:
I believe that's just a place where you can provide devise a secret key. Devise by default uses its own key.